### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values/strings.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#243]
```
translation to hebrew
```

Original issue reported on code.google.com by `eita...@gmail.com` on 1 Oct 2011 at 10:00

Attachments:
- [strings.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-243/comment-0/strings.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Looks great. Thanks!

Committed in r168.

I am unsure what localization code to use for this though. The ISO 639-1 
language code for Hebrew is "he", but the Android emulator uses "iw" as 
language code for Hebrew. Currently, it is committed as language code "iw".

Could you please verify that this works on an actual device with Hebrew 
localization?

I'm very glad to see the first RTL translation! Well done eitanbm. Please 
report any issues that you might see where we do not fully support RTL. I have 
a feeling that there are cases where we build strings programmatically which 
might fail.
```

Original comment by `oskar.wi...@gmail.com` on 7 Oct 2011 at 8:57
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.10**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
